.github/workflows/e2e.yaml

```
@@ -16,7 +16,7 @@ jobs:
     runs-on: "ubuntu-20.04"
```

```suggestion
    runs-on: "ubuntu-22.04"
```
There are similar changes are required the some files under `.github/workflows`.

---

Fixed.
https://github.com/topolvm/pvc-autoresizer/compare/efd79e388bcdb3289c6cb28b030bb991a4abb4f0..7593000fa645ee1ce0a1f58047e991f2b679ff7a